src/test/java/com/googleinterns/zoomtube/servlets/TranscriptServletTest.java

```diff
@@ -193,21 +194,21 @@ public void doGet_returnsLectureForLongVideoFromDatastore() throws ServletExcept
   @Test
   public void doPost_persistDataInDatastoreForLongVideo() throws ServletException, IOException {
     when(request.getParameter(LectureUtil.VIDEO_ID)).thenReturn(LONG_VIDEO_ID);
-    when(request.getParameter(LectureUtil.ID)).thenReturn(LECTURE_ID_C);
+    when(request.getParameter(LectureUtil.ID)).thenReturn(LECTURE_ID_C.toString());
 
     transcriptServlet.doPost(request, response);
 
-    int actualQueryCount = entitiesInDatastoreCount(LECTURE_ID_C);
-    int expectedQueryCount = (transcriptLines(LONG_VIDEO_JSON)).size();
-    assertThat(actualQueryCount).isEqualTo(expectedQueryCount);
+    int actualQuery = entitiesInDatastoreCount(LECTURE_ID_C);
```

Comment:
Why did you rename to actualQuery. It seems to me actualQueryCount was better because it indicates its a count.

Reply:
I think at the time, I considered count a filler word because int implies that it is a number. Looking at it again, it makes sense to switch it back to `actualQueryCount` since it being an `int` doesn't guarantee it's a count.

Done!

Reply:
I thought that might be the reason you removed it. Had you written actualQueryInt then I would have said int is filler, but count does describe what the number represents.